src/pudl/metadata/resources/eia.py

```diff
@@ -59,9 +56,18 @@
                 ],
             },
         },
-        "encoder": ENERGY_SOURCES_EIA,
+        "encoder": CODE_METADATA['energy_sources_eia'],
         "sources": ["eia860", "eia923"],
     },
+    "entity_types_eia": {
+        "description": "Descriptive labels for EIA entity type and ownership codes, taken from the EIA-861 form instructions, valid through 2023-05-31.",
+        "schema": {
+            "fields": ["code", "label", "description"],
+            "primary_key": ["code"]
+        },
+        "encoder": CODE_METADATA["entity_types_eia"],
+        "sources": ["eia861"]
+    },
```

Comment:
I added this into `RESOURCE_METADATA` since it wasn't included in the latest version of `dev` and `entity_types_eia` is now being included in the code metadata doc. Probably needs `foreign_key_rules`?

Reply:
Because the `entity_type` fields are still such a heterogeneous mess (#1392), I left them a free form string for now. It's going to take significant work to get them all on the same page, and that was beyond the scope of the issue I was working on. So we aren't using it as a coding table / foreign key yet.